wait-for-it.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2064,SC2206,SC2128
+# Use this script to test if a given TCP host/port are available
```

Comment:
Did you write this script or is it from somewhere else? If it's from somewhere else, we should add a reference to the original file and also mention the license of the original file.

Reply:
Yeah, I believe it is from:

https://github.com/vishnubob/wait-for-it

It's tagged with the MIT license.